Nakama/INClient.cs

```diff
@@ -74,6 +74,6 @@ public interface INClient
 
         void Send<T>(INCollatedMessage<T> message, Action<T> callback, Action<INError> errback);
 
-        void Send(INUncollatedMessage message, Action<bool> callback, Action<INError> errback);
+        void Send(INUncollatedMessage message, bool reliable, Action<bool> callback, Action<INError> errback);
```

Comment:
Would it be better to provide an overloaded method header so we maintain backwards compatibility? Not sure if it's worth it for such a simple code change to user's code?